tests/e2e/test_rosacli_cluster.go

```diff
@@ -1642,6 +1643,144 @@ var _ = Describe("HCP cluster creation negative testing",
 						ContainSubstring(
 							"ERR: Additional Allowed Principals is supported only for Hosted Control Planes"))
 			})
+
+		It("HCP cluster creation subnets validation - [id:72538]",
+			labels.High, labels.Runtime.Day1Negative,
+			func() {
+				if profile.ClusterConfig.Private {
```

Comment:
AFAIK, this test is HCP only

Reply:
Yes it is. Testing group is only for HCP negative.

Reply:
should be fine then